networkx/algorithms/Maximum Priority Matching/maximum_priority_matching.py

```diff
@@ -0,0 +1,2009 @@
+import doctest
```

Comment:
You should not make a folder for a single file. 

Put the maximum priority matching algorithm for general graphs in the main algorithms folder, and for bipartite graphs in  the bipartite folder.